package.json

```diff
@@ -21,38 +22,71 @@
     "styled-components": "5.1.1"
   },
   "devDependencies": {
+    "@babel/plugin-transform-react-jsx": "^7.10.4",
     "@fewlines/eslint-config": "2.1.1",
+    "@types/jest": "^26.0.4",
     "@types/node": "14.0.23",
     "@types/react": "16.9.43",
     "@types/react-dom": "16.9.8",
     "@types/styled-components": "5.1.1",
     "@typescript-eslint/eslint-plugin": "3.6.1",
     "@typescript-eslint/parser": "3.6.1",
+    "babel-jest": "^26.1.0",
     "babel-plugin-styled-components": "^1.10.7",
     "eslint": "7.4.0",
     "eslint-config-prettier": "6.11.0",
     "eslint-plugin-import": "2.22.0",
     "eslint-plugin-prettier": "3.1.4",
     "eslint-plugin-react": "7.20.3",
     "file-loader": "6.0.0",
+    "jest": "^26.1.0",
+    "jest-fetch-mock": "^3.0.3",
     "prettier": "2.0.5",
+    "ts-jest": "^26.1.2",
     "ts-node": "8.10.2",
     "typescript": "3.9.6",
     "url-loader": "4.1.0"
   },
   "babel": {
-    "presets": [
-      "next/babel"
-    ],
-    "plugins": [
-      [
-        "styled-components",
-        {
-          "ssr": true,
-          "displayName": true
-        }
-      ]
-    ]
+    "env": {
+      "development": {
```

Comment:
Should we bring back the `production` in this PR or let it in the next?

Reply:
I will add it as I need to do some more changes here.